### PR TITLE
Docs: Advanced variables section copy edit 

### PR DESCRIPTION
**Pull request by imatwawana**
Style/wording/grammar edits for topic `advanced-variables.md`

### CODE REVIEW

File: docusaurus/docs/advanced-variables.md [hunk 1]
```diff
@@ -3,22 +3,22 @@ id: advanced-variables
...
 
-Start with building a custom scene object that will display provided text.
+Start by building a custom scene object that will display provided text.
 
-This object:
+This object will:
 
-1. Will have a simple state that contains a string value(`text` property).
-2. Will render a textarea for state modifications and a preformatted text block for displaying the current value of the `text` state.
+1. Have a simple state that contains a string value(`text` property).
```
Comment (imatwawana) [inline docusaurus/docs/advanced-variables.md:20]: "string value(`text` property)" - should there be a space between value and the parentheses?

Comment (dprokop) [inline docusaurus/docs/advanced-variables.md:20]:
```suggestion
1. Have a simple state that contains a string value (`text` property).
```

File: docusaurus/docs/advanced-variables.md [hunk 2]
```diff
@@ -3,22 +3,22 @@ id: advanced-variables
 title: Variables in custom scene objects
 ---
 
-Variables lay a foundation for interactive dashboards. They allow dynamic configuration of what data is queried. Read more about [basic variables usage in Scenes](./variables.md).
+[Basic variables](./variables.md) lay the foundation for interactive dashboards. They allow dynamic configuration of which data is queried.
```
Comment (dprokop) [inline docusaurus/docs/advanced-variables.md:6]:
```suggestion
[Variables](./variables.md) lay the foundation for interactive dashboards. They allow dynamic configuration of which data is queried. 
```

Comment (dprokop) [reply docusaurus/docs/advanced-variables.md:6]: @imatwawana changed this back to Variables as here is no such a thing as Basic variables.

**Pull request merged**